script.js

```diff
@@ -28,50 +29,64 @@ $(document).ready(function () {
         writeNumbers = writeNumbers + '<div class="aside-game-board ">' + numbers[i] + '</div>';
     }
 
-    //create gameboard
+    //create game board
     for (var i = 0; i < colsCount; i++) {
         for (var j = 0; j < rowsCount; j++) {
             var fieldGameBoard = [
                 [i],
                 [j]
             ];
             var id = "field_" + i + "_" + j;
-            content = content + '<div id="' + id + '">' + '</div>';
-            console.log(fieldGameBoard);
+            content = content + '<div id="' + id + '" class="game-board-div">' + '</div>';
         }
     }
 
     //create ships
     var hitCount = 0;
 
     var gameBoardWithShips = [
-        [0,0,0,1,0,0,0],
-        [0,0,1,0,0,0,0],
-        [0,0,0,0,0,0,0],
-        [0,1,0,0,0,0,0],
-        [0,0,0,0,0,0,0],
-        [0,0,0,0,0,1,0],
-        [1,0,0,0,0,0,0]
+        [0, 0, 0, 1, 0, 0, 0],
+        [0, 0, 1, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0],
+        [0, 1, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 1, 0],
+        [1, 0, 0, 0, 0, 0, 0]
     ]
 
-    gameBoard.click(function(e){
+    //function that checks fields(empty or ship)
+    gameBoard.click(function (e) {
 
-        var coOrdinates = event.target.id.split('_');
-        // console.log(coOrdinates[1], coOrdinates[2]);
-        var shootResult = gameBoardWithShips[coOrdinates[1]][coOrdinates[2]];
-        if(shootResult === 1){
-            shootResult = 2;
-            hitCount++;  
-            e.target.style.background = "red";   
-            if(hitCount === 5){
-                setTimeout(function(){alert("WIN"); }, 10);
-            }
+        var coOrdinates = e.target.id.split('_');
+        var shotResult = gameBoardWithShips[coOrdinates[1]][coOrdinates[2]];
+        var target = $(e.target);
+        if (shotResult === 1) {
+            target.removeClass("game-board-div");
+            target.addClass("hit");
+            target.click(false);
+            hitCount++;
+        } else {
+            target.removeClass("game-board-div");
```

Comment:
Może lepsza nazwa dla tej klasy? Bo teraz sugeruje ze opisuje diva z polem a tak naprawde jest to styl  `dostepnego` pola.